autoload/maktaba/syscall.vim

```diff
@@ -67,6 +70,17 @@ function! s:DoSyscallCommon(syscall, CallFunc, throw_errors) abort
 endfunction
 
 
+" Compiles a dictionary describing the current vim state.
+function! s:CurrentEnv()
+  return {
+        \ 'tab': tabpagenr(),
```

Comment:
We indent line continuations by 4 (more than that leads to a lot of line wrapping).